renderer/components/timelines/status/Body.tsx

```diff
@@ -19,7 +19,7 @@ export default function Body(props: Props) {
       return (
         <div className="raw-html">
           <div
-            className="spoiler-text"
+            className="spoiler-text font-thin"
```

Comment:
Why do you use a thin font? Can you please share the image?

Reply:
IMHO, it looks nicer from information hierarchy point of view. But this may be a matter of personal taste.

With font-thin:

<img width="1029" alt="image" src="https://github.com/user-attachments/assets/e14bcef3-05f3-4a29-a0fb-155a1d8af163">


Without it: 

<img width="1029" alt="image" src="https://github.com/user-attachments/assets/ad4d75c4-f258-4e0e-996d-910d09f7b23a">

Reply:
[Cinny](https://cinny.in/) has done a good job of creating a highly legible Slack-like messaging UI which we can try to replicate:

<img width="1451" alt="image" src="https://github.com/user-attachments/assets/a66c5791-74f3-40f5-8841-02e0288f1ee0">

Reply:
Hm, looks good.